src/Inkluzitron/Data/Entities/User.cs

```diff
@@ -30,6 +31,29 @@ public class User
         public string KisNickname { get; set; }
         public DateTime? KisLastCheck { get; set; }
 
+        public string BirthdayDateImpl { get; set; } // 0000-00-00
```

Comment:
Proč takto? EF Core v SQLite neumí DateTime/Date?

Reply:
Mám pocit, že s tím byly nějaké problémy, ale vyzkouším.

Reply:
Tak to nakonec šlo bez problému, předěláno na obyčejné DateTIme.